### DISCUSSION TITLE
docs: clarify which versions of Renovate bot we support for how long

### DISCUSSION THREAD

**Discussion by HonkingGoose** [renovatebot/renovate#10056] [Suggest an Idea]
**EDIT:**

Pull request https://github.com/renovatebot/renovate/pull/10073 will add documentation about which version(s) of Renovate bot get official support. 

---

## Problem

Users are running Renovate version `24` instead of running the latest `25` releases.
Users will open bugs/features based on the old version.

The Renovate maintainers (including me!) will tell reporters to try again with the latest version to see if that fixes their problem.
This might be a bit unfriendly and confusing, if the bug reporter has a honest belief that version `24` is still supported?

There is no current documentation (that I could find) that explicitly states what the Renovate support scheme is.

## Current Renovate major version support

We're currently supporting `25` and `24` at once, I think?

I heard @rarkins say that we don't want to keep `24.x.x` around after Node 16 becomes current LTS...

## Proposal

Write down our support scheme, so that everybody is clear on what to expect.

The version support scheme could be any of those, to be decided on by the Renovate code wizards:

- Always use latest stream, no support for old stuff at all
- Support old Major version(s) for x amount of time after the next major release
- Support old Major version until version of Node that old major version supports is no longer current LTS
- Support old versions until deprecated with a deprecation commit/push to npmjs
- Something else

The FAQ seems a good place to put our "official Renovate supported version(s)" information.

**Comment by viceice** [comment]
I think we only support newest version. We only provide bugfixes to older versions if the hosted app needs to stay on an older major for a short time or  some critical bug needs to be fixed and the new major is blocked.
